plugins/hls-fourmolu-plugin/hls-fourmolu-plugin.cabal

```diff
@@ -29,7 +29,8 @@ library
   build-depends:
     , base            >=4.12 && <5
     , filepath
-    , fourmolu        ^>=0.3 || ^>=0.4 || ^>= 0.6 || ^>= 0.7
+    -- , fourmolu        ^>=0.3 || ^>=0.4 || ^>= 0.6 || ^>= 0.7
```

Comment:
Eugh, I take it we still can't drop support for old versions due to `stylish-haskell`: https://github.com/haskell/haskell-language-server/pull/2254#discussion_r721196303?

(if we could, we could obviously remove all the CPP)

Reply:
stylish-haskell has been updated since #2254  and I think it's very likely that this can be removed. Do you want to check in a follow-up PR?

Reply:
See https://github.com/haskell/haskell-language-server/pull/2950.